src/openms/source/CHEMISTRY/ModifiedPeptideGenerator.cpp

```diff
@@ -235,131 +227,48 @@ namespace OpenMS
         // Term specificity is ANYWHERE on the peptide, C_TERM or N_TERM
         // (currently no explicit support in OpenMS for protein C-term and
         // protein N-term)
+        // TODO This is not true anymore!
```

Comment:
I think the comment should say that if peptides are modified, there is no explicit support for protein C-term modifications (because the generator doesn't know if the peptide is terminal)

Reply:
yes but this could/should be added.

Reply:
Probably needs some interplay with digestion